packages/ember-toucan-core/src/components/form/fields/file-input.ts

```diff
@@ -10,16 +10,58 @@ export type FileEvent = (Event | MouseEvent) & { target: FileTarget | null };
 interface ToucanFormFileInputFieldComponentSignature {
   Element: HTMLInputElement;
   Args: {
+    /**
+     * Provide a string to this argument, a comma separated list of file types
```

Comment:
nit: The "Provide a string to this argument," part isn't really necessary, IMO. The type annotation lets us know it's a string, and we know these are all arguments.

```suggestion
     * A comma separated list of file types
```

(similar for the others below)

Reply:
I agree